### ISSUE TITLE
using decaying learning rate in tensorflow

### ISSUE THREAD

**Issue by MollyZhang** [MollyZhang/Multi-task-TCGA-learning#5]
https://www.tensorflow.org/versions/r0.12/api_docs/python/train/decaying_the_learning_rate

**Comment by MollyZhang** [comment]
Tried to lower AdamOptimizer learning rate from 1e-2 to 1e-3 to 1e-4 every 10,000 steps (batch size 100 at each step). The overall performance is WORSE than using 1e-2 or 1e-3 as the learning rate throughout, not sure why.
